Client/AdjustHelper.swift

```diff
@@ -34,10 +46,10 @@ final class AdjustHelper: FeatureFlaggable {
     /// Used to enable or disable Adjust SDK and it's features. We disable third party sharing by default.
```

Comment:
update the comment because we are not disabling by default anymore

Reply:
good catch, updated